source/_integrations/unifiprotect.markdown

```diff
@@ -94,6 +96,11 @@ Each UniFi Protect camera will get the following entities added:
 
 * **Camera** - A camera entity for each camera channel and RTSP(S) combination found for each camera (up to 6). Only the highest resolution RTSPS camera entity will be enabled by default.
 * **Media Player** - If your camera has a speaker, you will get a media player entity that allows you to play audio to your camera's speaker. Any audio file URI that is playable by FFmpeg will be able to be played to your speaker, including via the [TTS Say Service](https://www.home-assistant.io/integrations/tts/#service-say).
+* **Number** - The following number entities will be added for each camera:
```

Comment:
I really do think the documentation structure doesn't make sense.

People look at the information they can have access to and don't look them up by the platform.

Reply:
Okay.... do you have a suggestion on how to make it better?

Reply:
The problem is that the main chapter has devices/types that also are in this list here as platforms.

Now you end up with sensors that provide numbers and sensors that have cameras, and cameras having media players... the confusion here is that an end-user will have no clue in context between the difference of an actual device vs an entity platform.

I think it is too complex, to begin with. For example: `#### Entities`
While Home Assistant indeed has entities, and those drive the system, from a user perspective we work with devices & services. Entities are part of that system.

I think it makes more sense to talk in term of functionality.

For example; this integration supports UniFi protect cameras (and just that).
You could extend it with a more functional description of what can be controlled.

Reply:
I think I get what you mean. Should I refactor this PR for these changes (and the other ones you mentioned in previous reviews)? Or should I let this one go as is and do it all in a follow up?

Reply:
Let do a refactor afterward 👍